GuildedRoseLLC/Details/DetailsViewController.swift

```diff
@@ -2,6 +2,8 @@ import UIKit
 
 public class DetailViewController: UIViewController {
   
+    @IBOutlet public var quality: UILabel!
+    @IBOutlet public var sellIn: UILabel!
     @IBOutlet public var itemName: UILabel!
     public var setItem: (() -> Item)?
```

Comment:
I know this is old code, but I love having an explicit setter. It makes something like moving towards a protocol easier.